src/components/Editor/ItemEditor.vue

```diff
@@ -47,12 +47,43 @@
 
     <!-- item editor -->
     <div class="h-full border-2 rounded-t-xl mr-2 ml-2 p-2 item-editor-box">
-      <!-- question input box -->
+      <!-- question input box : expandable -->
       <input-text
         :placeholder="'placeholder'"
         :title="'Question'"
         v-model:value="questionText"
         ref="questionText"
+        class="p-2"
+        :type="{ boxType: 'textarea' }"
+      ></input-text>
+
+      <!-- time stamp input -- HH : MM : SS : mmm -->
+      <time-stamp-input
+        :title="'Time for popup (HH:MM:SS:mmm)'"
```

Comment:
- using a simpler text
- remove the format as the box placeholders already that
```suggestion
        :title="'Time for the question to appear'"
```